addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1492,6 +1493,183 @@ options.registry.gallery = options.Class.extend({
+                this.$target[0].dataset.progressBarStyle = 'none';
+                this.$target[0].dataset.shapeBackground = 'none';
+                break;
+        }
+        this.$target[0].dataset.layout = value;
+        this._refreshPublicWidgets();
+    },
+    /**
+    * Changes the URL the countdown redirects to once finished.
+    *
+    * @see this.selectClass for parameters
+    */
+    redirectUrl: function (previewMode, value, $opt) {
+        var val = $opt.find('input').val();
+        this.redirectUrlValue = val;
+        this.$target[0].dataset.redirectUrl = val;
+        this._refreshPublicWidgets();
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _setActive: function () {
+        this._super(...arguments);
+        this.$el.find('[data-display]').removeClass('active')
+            .filter('[data-display=' + this.$target[0].dataset.display + ']').addClass('active');
+        this.$el.find('[data-end-action]').removeClass('active')
+            .filter('[data-end-action=' + this.$target[0].dataset.endAction + ']').addClass('active');
+        this.$el.find('[data-layout]').removeClass('active')
+            .filter('[data-layout=' + this.$target[0].dataset.layout + ']').addClass('active');
+
+        this.$el.find('[data-end-time] > input').datetimepicker('date', moment(this.endTimeValue * 1000));
+        this.$el.find('[data-redirect-url] > input').val(this.redirectUrlValue);
+    },
+    /**
+     * @private
+     */
+    _updateUI: function () {
+        this._super(...arguments);
+        var dataset = this.$target[0].dataset;
+
+        // End Action UI
+        this.$el.find('[data-redirect-url]')
+            .toggleClass('d-none', dataset.endAction !== 'redirect');
+        this.$el.find('.toggle-edit-message')
+            .toggleClass('d-none', dataset.endAction !== 'message');
+        this.$el.find('[data-toggle-class="hide-countdown"]')
+            .toggleClass('d-none', dataset.endAction === 'nothing');
+
+        // End Message UI
+        this._updateUIEndMessage();
+
+        // Advanced UI
+        var showAdvanced = dataset.layout === 'advanced';
+        this.$el.find('[data-attribute-name="shape"]')
+            .toggleClass('d-none', !showAdvanced);
+        this.$el.find('[data-attribute-name="shapeBackground"], [data-attribute-name="progressBarStyle"]')
+            .toggleClass('d-none', !showAdvanced || dataset.shape === 'none');
+        this.$el.find('[data-attribute-name="progressBarWeight"]')
+            .toggleClass('d-none', !showAdvanced || dataset.progressBarStyle === 'none');
+    },
+    /**
+     * Shows or hides the countdown and its message depending of the chosen
+     * snippet options.
+     *
+     * @private
+     */
+    _updateUIEndMessage: function () {
+        var t = this.$target;
```

Comment:
Nan :p 
If you really want this, at least: `const $t`

Reply:
Alright, I will avoid shortcut as you don't like them.
Removed the shortcut and used `this.$target` (had to split the line).